metrics.go

```diff
@@ -9,6 +9,10 @@ import (
 	log "github.com/xlab/suplog"
 )
 
+func ReportFunc(fn, action string, tags ...Tags) {
+	reportFunc(fn, action, tags...)
+}
+
```

Comment:
Maxim helped to add custom report here, but adding a generic func is still fine https://github.com/InjectiveLabs/metrics/blob/master/reports.go#L3